.doc/tutorials/tutorial.rst

```diff
@@ -0,0 +1,220 @@
+
++------------------------------------------+------------------------------------------+
+| .. code-block:: yaml                     |                 |plot2|                  |
+|                                          |                                          |
+|    gnuplot_config:                       |                                          |
+|      plots:                              |                                          |
+|      - filters:                          |                                          |
+|        - ns: query/request               |                                          |
+|          val: jc                         |                                          |
+|          predicate: "="                  |                                          |
+|        - ns: query/collision_detector    |                                          |
+|          val: FCL                        |                                          |
+|          predicate: "="                  |                                          |
+|        - ns: query/scene                 |                                          |
+|          val: empty                      |                                          |
+|          predicate: "="                  |                                          |
+|                                          |                                          |
+|        # For better clarity              |                                          |
+|        - ns: query/planner               |                                          |
+|          val: RRTstar                    |                                          |
+|          predicate: "!="                 |                                          |
+|        labels:                           |                                          |
+|          - query/planner                 |                                          |
+|        metrics:                          |                                          |
+|        - name: time                      |                                          |
+|          type: boxplot                   |                                          |
+|      options:                            |                                          |
+|        terminal: QT                      |                                          |
+|                                          |                                          |
++------------------------------------------+------------------------------------------+
+
+
+We now want to compare the collision detector FCL against Bullet with the same parameters as above. This will double the previous boxplots because we are adding another collision detector and thus plot 12 boxplots (RRTstar is still filtered**. If this is too much information you can still filter out additionnal planners. Let's use the ``legends** parameter to bring out the comparision even more.
```

Comment:
```suggestion
We now want to compare the collision detector FCL against Bullet with the same parameters as above. This will double the previous boxplots because we are adding another collision detector and thus plot 12 boxplots (RRTstar is still filtered**. If this is too much information you can still filter out additional planners. Let's use the ``legends** parameter to bring out the comparison even more.
```